pkgs/by-name/mk/mkwebfont/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  mkwebfont,
+  rustPlatform,
+  testers,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mkwebfont";
+  version = "0.2.0-alpha10";
+
+  src = fetchFromGitHub {
+    owner = "Lymia";
+    repo = "mkwebfont";
+    rev = "v${version}";
+    hash = "sha256-EN2p8B5bU28p9lRT3uQlwp9DDIskVRvuIFA+M6qPCPY=";
+    fetchSubmodules = true;
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-hc1oD3JS4ZJM78SpnRQL65MdG3/y5cByDW0Xa3XXojc=";
+
+  passthru.tests.version = testers.testVersion { package = mkwebfont; };
+
+  meta = {
+    description = "Simple tool for turning .ttf/.otf files into webfonts";
+    mainProgram = "mkwebfont";
+    homepage = "https://github.com/Lymia/mkwebfont";
+    changelog = "https://github.com/Lymia/mkwebfont/raw/v${version}/RELEASES.md";
```

```suggestion
    changelog = "https://github.com/Lymia/mkwebfont/blob/${finalAttrs.src.rev}/RELEASES.md";
```
1. Usually web-rendered changelogs are preferred
2. `src.rev` is a bit more reliable (for cases when you need to package a commit outside of the tag); the src will have that attr populated anyway